lib/main.coffee

```diff
@@ -198,6 +198,7 @@ module.exports =
         if text.indexOf("\n") is -1
           imageToken = if isImage then '!' else ''
           if text.match(/[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/)
+            if text.match(/\.(jpg|jpeg|png|gif|tiff)/) then imageToken = '!'
```

Comment:
Not entirely sure this is a good thing. If I'm reading this right, this is assuming that if I use the `@` shortcut on a link that looks like an image, that it will create an image link and not a regular link, right?